plugin/bookmark.vim

```diff
@@ -253,7 +253,9 @@ function! CallDeprecatedCommand(fun, args)
   return call(Fn, a:args)
 endfunction
 
-function! BookmarkModify(diff)
+function! BookmarkModify(diff, ...)
```

Comment:
Thanks for you the explaination. I would really like to have this better readable. I suggested below to have a second exposed function that would internally call the same method.